### DISCUSSION TITLE
如何启用llama模型？

### DISCUSSION THREAD

**Discussion by HuengchI** [GaiZhenbiao/ChuanhuChatGPT#591] [Q&A]
请问如何启用llama模型？

**Comment by GaiZhenbiao** [comment]
安装requirements_advanced.txt之后，在下拉列表中选择llama即可

**Reply by HuengchI** [reply]
您好，我安装完之后，出现了`Tokenizer class LLaMATokenizer does not exist or is not currently imported.`错误，应该怎么办呢？

**Reply by GaiZhenbiao** [reply]
看llama的一个issue，我此前回答过这个问题了。

**Reply by HuengchI** [reply]
感谢！我把[相关issue](https://github.com/GaiZhenbiao/ChuanhuChatGPT/issues/655#issuecomment-1509063338) link到这里，供后人查看。

这个问题解决之后，出现错误：`ProcessGroupNCCL is only supported with GPUs, no GPUs found!`。请问如何仅使用CPU运行llama模型呢？
